gcloud-java-datastore/src/main/java/com/google/cloud/datastore/testing/LocalDatastoreHelper.java

```diff
@@ -570,6 +570,20 @@ public DatastoreOptions options() {
   }
 
   /**
+   * Returns a {@link DatastoreOptions} instance that sets the host to use the Datastore emulator on
+   * localhost. The default namespace is set to {@code namespace}.
+   */
+  public DatastoreOptions options(String namespace) {
+    return DatastoreOptions.builder()
```

Comment:
Maybe the  two `options()` method could delegate to an internal optionsBuilder method for the common settings?